recipes/coincurve/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "coincurve" %}
+{% set version = "18.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c86626afe417a09d8e80e56780efcae3ae516203b23b5ade84813916e1c94fc1
+  patches:
+    - resolve_abs_path_SOURCES.txt.patch  # [not win]
+    - update_pyproject.toml.patch
+    - update_setup.py.patch  # [win]
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/libc.so.6
+  script_env:
+    - COINCURVE_UPSTREAM_REF=ddf2b2910eb19032f8dd657c66735115ae24bfba
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pip
+    - python
+    - python-build
+    - requests
+    - setuptools
+    - wheel
+    - autoconf  # [not win]
+    - automake  # [not win]
+    - libtool  # [not win]
+    - m4  # [not win]
+    - make  # [not win]
+    - pkg-config  # [not win]
+    - asn1crypto  # [win]
+    - m2-bash  # [win]
+    - m2-autoconf  # [win]
+    - m2-automake-wrapper  # [win]
+    - m2-gzip  # [win]
+    - m2-libtool  # [win]
+    - m2-m4  # [win]
+    - m2-pkg-config  # [win]
+    - m2-sed  # [win]
+    - m2w64-gcc  # [win]
+    - m2w64-make  # [win]
+    - posix  # [win]
+    - pydflow  # [win]
```

Comment:
I would suggest using msvc toolchain instead of m2 if you have the option. The m2 conda toolchain isn't maintained well.

Reply:
Could you point me to documentation on the `msvc` alternative, is it what is referred to as `msys2`. This is only for personal education, based on your review, it looks like the latter work I did on coincurve is preferred.

Would you advise to close this PR?

Reply:
By MSVC, I mean Microsoft Visual Studio which is the default compiler for the Windows platform. The Visual Studio compiler is provided by the `{{ compiler('c') }}` package. Whether or not this project will allow you to choose different compilers is up to the project.

Reply:
> I do use MSVC, but I need all these in order to make the old make-based build work.

OK.

> You're moving too fast for me, I don't want to waste your time, but I also want to understand what I do wrong. You closed the PR that I thought was the better one, so I am quite confused

There is no way to transfer review comments from one PR to another. I've already started reviewing this package in this PR. It should be trivial for you to merge your other branch into this one.

Reply:
Separated `libsecp256k1` which builds using CMake. This is resolved